mcp/gcp_cloudrun.tf

```diff
@@ -0,0 +1,112 @@
+
+//noinspection HILUnresolvedReference
+data "google_project" "default" {
+  project_id = lookup(local.cloudrun, "project_id", "")
+}
+
+data "google_container_registry_image" "default"{
+  name = lookup(local.cloudrun_specs, "image_name", "")
+  project = data.google_project.default.project_id
+}
+
+//noinspection HILUnresolvedReference
+resource "google_cloud_run_service" "self" {
+  count = local.cloudrun == {} ? 0 : 1
```

Comment:
like what is mentioned in the [structure](https://github.com/mesoform/terraform-infrastructure-modules/tree/master/mcp#structure), there will often be considerations for this to be a collection of micro services. Look to use for_each and follow the pattern for GAE